### PR TITLE
add queryToHeader filter

### PR DESCRIPTION
**Pull request by szuecs**
queryToHeader("q", "X-Q") will copy the value of q param to the value of X-Q header.

The optional 3rd argument can be used to prepend the value, for example to add "Bearer ".

### CODE REVIEW

File: docs/filters.md [hunk 1]
```diff
@@ -727,3 +727,28 @@ headerToQuery("X-Foo-Header", "foo-query-param")
...
+
+* The name of the query param key to pick from request
+* The name of the header to add to request
+* The static prefix string to prepend to the header value
+
+Examples:
+
+```
+headerToQuery("foo-query-param", "X-Foo-Header")
+headerToQuery("access_token", "Authorization", "Bearer ")
```
Comment (mikkeloscar) [inline docs/filters.md]: This should be `queryToHeader`

Comment (mikkeloscar) [reply docs/filters.md]: I also think the prepending of the value is a bit magic, would it make sense to make it a format string? i.e.:

```
queryToHeader("access_token", "Authorizaton", "Bearer %s")
```

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
@mikkeloscar I changed the code to make it a format string, cool idea!

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**